app/data_models/fulfilment_request.py

```diff
@@ -20,7 +25,7 @@ def message(self) -> bytes:
                 "source": "QUESTIONNAIRE_RUNNER",
                 "channel": "EQ",
                 "dateTime": datetime.now(tz=tzutc()).isoformat(),
-                "transactionId": str(uuid4()),
+                "transactionId": self.transactionId,
```

Comment:
Other than this dict property, every other occurrence of `transactionId` should be `transaction_id`.

Reply:
Done